lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -876,6 +876,19 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
 
     return b.create<arith::SelectOp>(loc, mask, fillValue, input);
   }
+  if (auto maskedFillTensor = dyn_cast<AtenMaskedFillTensorOp>(op)) {
+    AtenMaskedFillScalarOp::Adaptor adaptor(operands);
+    Type dtype = converter->convertType(maskedFillTensor.getType())
+                     .cast<RankedTensorType>()
+                     .getElementType();
+
+    Value input = payloadArgs[0];
+    Value mask = payloadArgs[1];
+    Value scalarValue =
+        b.create<tensor::ExtractOp>(loc, adaptor.value(), ValueRange({}));
```

Comment:
The `linalg.generic` function created for the op should also be "iterating" over the fill value tensor, so you don't need to extract here. You should be able to do `payloadArgs[2]`

Reply:
Done.

Reply:
Thanks!